vision/emergent_object/emergent_object.py

```diff
@@ -0,0 +1,61 @@
+import torch
+
+# You may have to install:
+#   pandas
+#   torchvision
+#   tqdm
+#   seaborn
+
+MODEL_PATH = "vision/emergent_object/best.pt"
+
+
+# Function to do detection / classification
+def emergent_object_detection(image, model):
+    # Convert to RGB
+    image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
```

Comment:
Type hint for `image` should be `Image` type alias from vision > common > constants.py